### DISCUSSION TITLE
nspanel  problem with Look of the Button in Buttonpage 1 eg

### DISCUSSION THREAD

**Discussion by Lucer61** [Blackymas/NSPanel_HA_Blueprint#1202] [Q&A] [answered]
Hello community
I have now the Nextion display working and flashed it by Rx and Tx from the Nextion Editor.  
Blueprint is running, the homescreen is ok, all data is displayed but now it comes the ButtonPage 1, 
I create a button here, it is not displayed correctly, shouldn't Homeassistant choose the correct display by the entity?  
I just don't have the right Look of the Button . 
I only see a little icon that cant make action
 Do you have a tip? 
 I would have liked to attach a screenshot, unfortunately this is not possible here

**Comment by edwardtfn** [comment]
> I just don't have the right Look of the Button .

Could you please share a picture?

Also, when you restart your panel, which versions are shown for TFT, ESPHome and Blueprint on the blue boot screen?

> I would have liked to attach a screenshot, unfortunately this is not possible here

Are you sure? I have the option to add pictures bellow the box where I'm writing.

![IMG_6516](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/94725493/7cf7b374-f714-41f8-a124-defc47c72752)

**Comment by Lucer61** [comment]
thanks for help me!
bootup shows  TFT vers. 3.4  ESPHome vers. 4.0.2
below the screenshot of my homeassistant and NSPanel.
I hope you can see the screenshots.
![IMG_2760](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/46453406/2e90e90d-be05-4169-af1a-22f709e3462e)
![IMG_2759](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/46453406/9c8381ee-f2ad-4f85-83c3-89c1dc06b307)
![IMG_2758](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/46453406/e1b53825-7315-48db-8aee-07505ccb3faf)
![IMG_2757](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/46453406/0722b2f0-c757-4c8b-bed2-b18c3979bd71)

**Reply by edwardtfn** [reply, marked as answer]
The error in the button page is caused due to the version mismatch. You must have everything on v3.4 or v4.0.x.

https://github.com/Blackymas/NSPanel_HA_Blueprint/wiki/(EN)-HowTo---All-important-thing-you-should-know#2-update-blueprint

**Comment by Lucer61** [comment]
thank you very much for the fast help, everithing ist fine now!
all vers. now at 4.0.2  mistake at TFT was old vers. 
greatfull !

![IMG_2765](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/46453406/00b5dfa1-a58b-42c9-846c-0fee428b5fdc)
![IMG_2764](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/46453406/9b21cfa9-b583-481b-8e87-4e4ac909aa81)

**Comment by Lucer61** [comment]
finished resolved!
sorry I couldend see the button for close
